fala/apps/adviser/tests/test_crown_dependencies_view.py

```diff
@@ -6,7 +6,10 @@
 @override_settings(FEATURE_FLAG_NO_MAP=True)
 class PostcodeValidationTest(SimpleTestCase):
     client = Client()
-    url = reverse("search")
+
```

Comment:
This no longer seems necessary if we're not varying the urlconf, but do we like it anyway?